components/http_client.rst

```diff
@@ -362,30 +365,26 @@ regular expression applied to relative URLs::
             'base_uri' => 'https://api.github.com/',
             // ...
         ],
-
-        '*' => [
-            // ...
-        ]
     ],
         // this is the regexp applied to all relative URLs
         'https://api\.github\.com/'
     );
 
-PSR-7 and PSR-18 Compatibility
-------------------------------
+PSR-18 Compatibility
+--------------------
 
-This component uses its own interfaces and exception classes different from the
-ones defined in `PSR-7`_ (HTTP message interfaces) and `PSR-18`_ (HTTP Client).
-However, it includes the :class:`Symfony\\Component\\HttpClient\\Psr18Client`
+This component uses and implements abstractions defined by the
+``symfony/contracts`` package. It also implements the `PSR-18`_ (HTTP Client)
+specifications via the :class:`Symfony\\Component\\HttpClient\\Psr18Client`
 class, which is an adapter to turn a Symfony ``HttpClientInterface`` into a
 PSR-18 ``ClientInterface``.
 
-Before using it in your application, run the following commands to install the
-required dependencies:
+To use it, you the ``psr/http-client`` package and use a `PSR-17`_
```

Comment:
This reads weird -> `To use it, you the [...]`  I think the original phrase was better in this case.